.github/workflows/merge-pr.yml

```diff
@@ -47,7 +26,7 @@ jobs:
       - name: Check Out Repo
         uses: actions/checkout@v3
       - name: Install Rust Toolchain
-        uses: actions-rs/toolchain@v1
+        uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af
```

Comment:
While auditing these, I think we need to switch off of this action to https://github.com/marketplace/actions/rustup-toolchain-install

It is actually maintained and looks like it is the go to replacement.